### ISSUE TITLE
Set Origin to Selected Does not Install in 3.1.2

### ISSUE THREAD

**Issue by HarleyInfinity** [ChrisCrossCrash/SetOriginInEditMode#2]
After installing the script it does not appear in the addon list despite being copied to the addon directory on disk.

**Comment by ChrisCrossCrash** [comment]
@HarleyInfinity Sorry for not getting back to you sooner. Could you get it figured out?

It works for me on 3.1.2. Perhaps you are in the "Official" tab when you are looking at the addon menu?

![image](https://user-images.githubusercontent.com/40023417/164532618-bca9bf12-c2a8-4784-b9fa-5a19fe028aef.png)

**Comment by HarleyInfinity** [comment]
Yes, it's the oddest thing. I've looked under community but it just never appears. I tried today on another PC just in case but I get the same issue. :(

**Comment by ChrisCrossCrash** [comment]
@HarleyInfinity sorry again for the delay. I've just moved to a whole different country, so I was a little distracted.

I've just tried installing it as described in README.md and it worked on a different computer with a fresh install of Blender 3.1.2 and it worked for me. Here's how I did it:

1. Downloaded [`set_origin_in_edit_mode.py`](https://github.com/ChrisCrossCrash/SetOriginInEditMode/blob/master/set_origin_in_edit_mode.py) to my desktop.
2. In Blender, I went to Preferences > Add-ons > Install and selected the python file on my desktop.
3. Clicked the checkbox next to the add-on in the list.

I'm not sure why it wouldn't work. Perhaps you accidentally saved the file with a `.txt` file extension?
